gdal/ogr/ogrsf_frmts/flatgeobuf/packedrtree.h

```diff
@@ -0,0 +1,75 @@
+#ifndef FLATGEOBUF_PACKEDRTREE_H_
```

Comment:
non-generated file, right ? license ?

Reply:
Right, LICENSE file added to containing dir. (ICS license)